examples/custom_potential.jl

```diff
@@ -7,44 +7,37 @@
 using DFTK
 using LinearAlgebra
 
-# First, we define a new element which represents a nucleus generating a custom
-# potential
-struct ElementCustomPotential <: DFTK.Element
-    pot_real::Function      # Real potential
-    pot_fourier::Function   # Fourier potential
+# First, we define a new element which represents a nucleus generating
+# a Gaussian potential.
+struct ElementGaussian <: DFTK.Element
+    α  # Prefactor
+    L  # Width of the Gaussian nucleus
 end
 
-# We need to extend two methods to access the real and Fourier forms of
-# the potential during the computations performed by DFTK
-function DFTK.local_potential_fourier(el::ElementCustomPotential, q::Real)
-    return el.pot_fourier(q)
+# We extend the two methods providing access to the real and Fourier
+# representation of the potential to DFTK.
+
+function DFTK.local_potential_fourier(el::ElementGaussian, q::Real)
+    el.α * exp(- (q * el.L)^2 / 4)
```

Comment:
Normalizations look fishy here, there's no pi?

Reply:
Yes I think they are, but I copied from Gaspard ...

Reply:
Let me take a look.

Reply:
The convention we use is V(q) = int_{R^3} V(x) e^{-iqx} dx (documented in elements.jl) so very likely there's a factor missing

Reply:
@gkemlin :-)